### PR TITLE
Fixes #1399 edits to glomerular visceral epithelial cell (fresh branc…

### LINKED ISSUE #1399: glomerular visceral epithelial cell should be part of glomerular visceral epithelium
it is currently 

subClassOf part of some visceral layer of glomerular capsule

This should be changed to 

EquivalentTo 'epithelial cell'  and 'part of' some 'glomerular visceral epithelium'

Closely related ticket: https://github.com/obophenotype/uberon/issues/2230

### PR DESCRIPTION
**Pull request by bvarner-ebi**
…h- replaces #1401)

Fixes #1399 edits to glomerular visceral epithelial cell (fresh branch- replaces #1401)

Since there were several updates (merges, obsoletes, imports, new releases), it became burdensome to manage the branch in PR #1401. Please use this new pull request and branch to continue from #1401.

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -9027,6 +9027,7 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000653 "podocyte"^^xsd:stri
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0000653 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:comment obo:CL_0000653 "GO has podocyte as a related synonym due to concern this may refer to more than one cell type. However, this did not turn up in my search. If podocyte does refer to more than one cell type, then we should classify the synonym to 'related'[tfm]."^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000653 "glomerular visceral epithelial cell"^^xsd:string)
+EquivalentClasses(obo:CL_0000653 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005751)))
 SubClassOf(obo:CL_0000653 obo:CL_0002518)
 SubClassOf(obo:CL_0000653 obo:CL_0002522)
 SubClassOf(obo:CL_0000653 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005751))
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:9033]: remove redundant subclass - already in equiv class

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -9027,6 +9027,7 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000653 "podocyte"^^xsd:stri
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0000653 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:comment obo:CL_0000653 "GO has podocyte as a related synonym due to concern this may refer to more than one cell type. However, this did not turn up in my search. If podocyte does refer to more than one cell type, then we should classify the synonym to 'related'[tfm]."^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000653 "glomerular visceral epithelial cell"^^xsd:string)
+EquivalentClasses(obo:CL_0000653 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005751)))
 SubClassOf(obo:CL_0000653 obo:CL_0002518)
```
[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:9031]: Think it is inferred - epithelial cell of glomerular capsule (which is inferred) is subclass of kidney glomerular epithelial cell which is subclass of kidney epithelial cell

[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:9031]: PS therefore this axiom is redundant too

**Review by shawntanzk — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl (hunk 2 above)
[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:9031]: btws, if you click on the ? on subclass of kidney epithelial cell axiom, you can see this: 
Screenshot 2022-02-22 at 13 15 21

[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:9031]: Thanks for reviewing, @shawntanzk. So it is clear, are you saying
**subClass of 'kidney epithelial cell'** 
is redundant and can be removed?

[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:9031]: yes :)

**Review by shawntanzk — APPROVED** [review]

**Comment by bvarner-ebi** [comment]
Thanks for reviewing, @shawntanzk, and thanks for all the imports and release updates you did that facilitated this edit!

**Pull request merged**